src/makeRobotAccountant.js

```diff
@@ -4,9 +4,26 @@
  *
  * @return {function}
  */
-
 function makeRobotAccountant() {
-  // write code here
-}
+  let session = 0;
```

Comment:
I guess it's better to name it `count` or `calls`, so it's clear that it counts something